views/js/qtiCreator/widgets/static/math/states/Active.js

```diff
@@ -357,7 +357,7 @@ define([
         });
 
         if(!$mathField.data('$tooltip')){
-            widgetTooltip = tooltip.error($mathField, $tooltipContent, {
+            widgetTooltip = tooltip.error($mathField, $tooltipContent[0].outerHTML, {
```

Comment:
Shouldn't it be only : 
`var tooltipContent = __('Currently conversion from MathML to LaTeX is not available. Editing MathML here will have the LaTex code discarded.'),`
and
 `widgetTooltip = tooltip.error($mathField, tooltipContent, {`

I don't understand why there's a wrapping `<span>`

Reply:
Checked, you are right, there is no need to have DOM element as a message. Fixed